omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -27,21 +29,27 @@
 public class MarkPatientDeadPageController {
     protected final Log log = LogFactory.getLog(this.getClass());
 
-    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") String patientId) {
+    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") Patient patient) {
```

Comment:
Was changing from parameter String patientId to Patient patient required as part of this ticket?

Reply:
@dkayiwa It was not. But With String It could only take uuid which was breaking the edit option in patient status suggested by the ticket. Thats why it was changed.

Reply:
@slubwama did you see the above comment?

Reply:
@dkayiwa Yes and this is the work on the comment
Ticket https://issues.openmrs.org/browse/RA-1504
PR https://github.com/openmrs/openmrs-module-coreapps/pull/150

Reply:
So can you remove those changes from this pull request?

Reply:
@slubwama did you see the above comment?

Reply:
@slubwama did you see the above comment?